CallAutomation_AppointmentReminder/CallAutomation_AppointmentReminder/Properties/launchSettings.json

```diff
@@ -0,0 +1,31 @@
+﻿{
+  "$schema": "https://json.schemastore.org/launchsettings.json",
+  "iisSettings": {
+    "windowsAuthentication": false,
+    "anonymousAuthentication": true,
+    "iisExpress": {
+      "applicationUrl": "http://localhost:43269",
+      "sslPort": 44364
+    }
+  },
+  "profiles": {
+    "CallAutomation_AppointmentReminder": {
+      "commandName": "Project",
+      "dotnetRunMessages": true,
+      "launchBrowser": true,
+      "launchUrl": "swagger",
+      "applicationUrl": "https://localhost:7196;http://localhost:5196",
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
```

Comment:
we can use the new dev tunnels feature in here: https://learn.microsoft.com/en-us/connectors/custom-connectors/port-tunneling